docker/main/Dockerfile

```diff
@@ -4,14 +4,16 @@ COPY dist /app
 
 WORKDIR /app
 
-RUN ls -lrt /app
-RUN mkdir -p /etc/appcatalog
-RUN chown -R nobody:nogroup /app && chmod +x /app/mycompany.sh
-RUN chown -R nobody:nogroup /app && chmod +x /app/appcatalog.sh
-RUN chown -R nobody:nogroup /app && chmod +x /app/appcatalog-configuration
-RUN chown -R nobody:nogroup /app && chmod +x /app/appcatalog
+RUN chown -R nobody:nogroup /app \
+    && chmod +x /app/mycompany.sh \
+                /app/appcatalog.sh \
+                /app/appcatalog-configuration \
+                /app/appcatalog \
+    && apt-get update \
+    && apt-get install -y curl wget ca-certificates
+
 USER nobody
 
 EXPOSE 8081
 
-CMD ./appcatalog-configuration --debug --output=./config.json start && ./appcatalog.sh --config=./config.json
+CMD ./appcatalog-configuration --debug --output=./config.json start && ./appcatalog --config=./config.json
```

Comment:
why did you remove .sh ... it's for loading mycompany.sh if needed for one ENV ?

Reply:
It contains a bug. The "/api" route is appearing twice in the appcatalog.sh file. I preferred deactivating it temporarily.